pydis_site/apps/content/templatetags/page_github.py

```diff
@@ -0,0 +1,21 @@
+import requests
+from django import template
+
+register = template.Library()
+
+
+@register.filter()
+def page_github(current_uri: str) -> str:
+    """
+    Return the corresponding GitHub page for the current site page.
+
+    Return a directory if the page is a directory on GitHub otherwise return the Markdown file
+    """
+    github_uri = current_uri.replace(
+        "http://pythondiscord.local:8000/pages/",
```

Comment:
I don't think this would work if we were using another URI, like `https://pythondiscord.com/`.

Reply:
Oh right